dask_ml/model_selection/_search.py

```diff
@@ -326,7 +330,7 @@ def do_fit_and_score(
     scorer,
     return_train_score,
 ):
-    if not isinstance(est, Pipeline):
```

Comment:
Does `imblearn.pipeline.Pipeline` not subclass `sklearn.pipeline.Pipeline`?

Reply:
We do.